app/src/main/res/values/strings.xml

```diff
@@ -287,30 +283,29 @@
 
     <!-- Notifications -->
     <string name="updated_matches_notification_title">
-        Woohoo! Beeline has new matches for you!
+        Psst! Beeline has new matches for you!
     </string>
     <string name="updated_matches_notification_text">
-        Once a day, Beeline runs an algorithm to pair the hotties of the world. The algorithm just
-        finished running. You\'ve got some hot peeps to check out!
+        Once a day, Beeline runs an algorithm to pair you beautiful people. The algorithm just
+        finished running. You\'ve been matched!
     </string>
 
     <string name="only_lost_matches_notification_title">
         You\'ve had some matches in Beeline. Now they are gone.
     </string>
     <string name="only_lost_matches_notification_text">
-        Once a day, Beeline runs an algorithm to pair the hotties of the world. The last person
-        just lost interest in you. But fret not. There are many things you can do. (1) Rank more
-        users to find the special one who\'s into you. (2) Text your matches faster. Once you sent
-        the first message, the match becomes locked. (3) This is the hardest yet most satisfying.
-        Become more attractive. Hit the gym. Get new clothes. Put a smile on your face by following
-        your passion.
+        Once a day, Beeline runs an algorithm to pair you beautiful people. The last person
+        just lost interest in you. But don't worry. There are many things you can do. (1) Rank more
+        users to find the special one who\'s into you. (2) Text your matches faster. Once you send
+        the first message, the match becomes locked. (3)Become more attractive. Put a smile on your face
+        by following your passion.
     </string>
 
     <string name="new_message_notification_title">
         New message!
     </string>
     <string name="new_message_notification_text">
-        Check out what a cutie is writing you!
+        Check out what someone's writing you!
```

Comment:
Isn't a sweet name a lot more exciting than the generic someone? "Hottie alert" gets me a lot more giddy than "Alert." (Not saying to use the word hottie specifically.)